soc/arm/atmel_sam0/common/soc_gpio.h

```diff
@@ -0,0 +1,125 @@
+/*
+ * Copyright (c) 2016-2017 Piotr Mienkowski
+ * Copyright (c) 2020-2021 Gerson Fernando Budke
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/** @file
+ * @brief Atmel SAM0 MCU family I/O Pin Controller (PORT)
+ */
+
+#ifndef _ATMEL_SAM0_SOC_GPIO_H_
+#define _ATMEL_SAM0_SOC_GPIO_H_
```

Comment:
Let's change it to `ATMEL_SAM0_SOC_GPIO_H_`. That's copied from SAM family where it's not yet fixed. However, identifiers starting with an underscore and a capital letter are reserved by C, we shouldn't use them.